publish-common/src/main/java/com/ericsson/eiffel/remrem/publish/helper/RabbitMqProperties.java

```diff
@@ -279,7 +274,8 @@ public void createRabbitMqConnection() throws RemRemPublishException {
             }
         } catch (IOException | TimeoutException e) {
             log.error(e.getMessage(), e);
-            throw new RemRemPublishException("Failed to create connection for Rabbitmq ::" + factory.getHost() + ":" + factory.getPort(), e);
+            throw new RemRemPublishException("Failed to create connection for Rabbitmq ::", factory,
```

Comment:
I'd add a space between :: and hostname to make the message easier to read.

Reply:
changed